blueprints/Muayad.py

```diff
@@ -0,0 +1,116 @@
+from flask import Blueprint, jsonify, request
+
+muayad = Blueprint("muayad", __name__, url_prefix="/mw")
+
+def Sum(my_list: list):
+    total_sum = sum(my_list)
+    print(f"You are going to add the array {my_list} together!!!!!!!")
+    return total_sum
+
+@muayad.route("/sum", methods=["GET","POST"])
+def sum_flask():
+    if request.method == "GET":
+        values = request.args
+    else:
+        values = request.json
+
+    my_list = values.get("my_list")
+    if my_list is None:
+        return jsonify(error="Missing 'my_list' parameter"), 400
+    my_list = [int(num) for num in my_list.split(',')]
```

Comment:
you need to make sure that the number doesn't have spaces by doing `strip()` before casting to `int`